index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav>
+    <ul class="menu">
+        <div>
+          <li class="logo">My Logo</li>
+        </div>
+        <div class="menu2">
+          <li class="item"><a href="#">Portfoilo</a></li>
+          <li class="item"><a href="#">about</a></li>
+          <li class="item"><a href="#">contact</a></li>
+          <li class="toggle"><a href=""><img src="./images/Icon - Menu.png" alt=""></a></li>
+        </div>
+    </ul>
+</nav>
+  <section>
+    <ul class="section1">
+        <li class="item1 h1">I’m Kirubel. <br>Glad to see you!</li>
+        <li class="item1 p"><p>I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,
+            don’t hesitate to contact me.</p></li>
+        <li class="item1" >LET'S CONNECT</li>
+        <div class="social">
+          <li class="item1"><a href="#"><img src="./images/twitter.png" alt=""></a></li>
+          <li class="item1"><a href="#"><img src="./images/Linkedin icon.png" alt=""></a></li>
+          <li class="item1"><a href="#"><img src="./images/medium.png" alt=""></a></li>
+          <li class="item1"><a href="#"><img src="./images/Vector (1).png" alt=""></a></li>
+          <li class="item1"><a href="#"><img src="./images/angellist.png" alt=""></a></li>
```

Comment:
Please include an `alt` text for each of your icons. It's important to always include that because it helps with your website's accessibility for screen readers and also displays on the page if the image/icon fails to load.